tests/Integration/Momento.Sdk.Incubating.Tests/DictionaryTest.cs

```diff
@@ -97,9 +98,9 @@ public async Task DictionarySetGetAsync_FieldIsByteArrayValueIsByteArray_Refresh
         var field = Utils.NewGuidByteArray();
         var value = Utils.NewGuidByteArray();
 
-        CacheDictionarySetResponse setResponse = await client.DictionarySetAsync(cacheName, dictionaryName, field, value, false, ttl: TimeSpan.FromSeconds(2));
+        CacheDictionarySetResponse setResponse = await client.DictionarySetAsync(cacheName, dictionaryName, field, value, CollectionUpdateTtl.fromTimespan(TimeSpan.FromSeconds(2)));
```

Comment:
I keep having to look back at what the default refresh strategy is for `fromTimespan`. How about we make it explicit?

```suggestion
        CacheDictionarySetResponse setResponse = await client.DictionarySetAsync(cacheName, dictionaryName, field, value, CollectionTtls.onCreateSet(TimeSpan.FromSeconds(2)).onUpdateUseDefault())
```